backend/codesirius/problems/models/language.py

```
@@ -13,12 +13,17 @@ class Language(BaseModel):
-    version = models.CharField(max_length=50, blank=True)  # Optional
+    name = models.CharField(max_length=50)
+    version = models.CharField(max_length=50)  # Optional
 
     class Meta:
         verbose_name = "Language"
         verbose_name_plural = "Languages"
+        constraints = [
+            models.UniqueConstraint(
+                fields=["name", "version"], name="unique_language_name_version"
+            )
+        ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)